lib/paths/index.ts

```diff
@@ -67,11 +76,12 @@ type RecursivePaths<
                           UserOptions,
                           {
                             depth: Append<CallOptions["depth"]>;
+                            anyArrayIndexAccessor: AnyArrayIndexAccessor<Value, UserOptions>;
```

Comment:
As per comment above, `anyArrayIndexAccessor` is static, I suggest removing this line:

```suggestion
```